packages/compass-e2e-tests/index.ts

```diff
@@ -244,6 +295,9 @@ async function run() {
 
     cleanup();
   }
+
+  // TODO: why does this somehow hang? How do you debug that?
+  process.exit(process.exitCode ?? 0);
```

Comment:
[`why-is-node-running`](https://www.npmjs.com/package/why-is-node-running) is a good starting point to investigate stuff like this

Reply:
Oof it had an error while trying to copy logs when stopped that was swallowed and I never noticed. So it never actually stopped compass properly.